digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/RoadAddressLinkBuilder.scala

```diff
@@ -38,6 +38,24 @@ object RoadAddressLinkBuilder extends AddressLinkBuilder {
 
   }
 
+  def buildSimpleLink(roadAddress: RoadAddress): RoadAddressLink = {
```

Comment:
maybe one simple test for this method?

Reply:
Might be too much overhead as it's only a really simple mapping.